apps/web/src/pages/Users/IndexUserPage/IndexUserPage.tsx

```diff
@@ -44,7 +50,38 @@ export const IndexUserPage = () => {
         crumbs={navigationCrumbs}
       />
       <AdminContentWrapper table>
-        <UserTable title={formattedPageTitle} />
+        <UserTable title={formattedPageTitle} newSearch={!isOldSearchChecked} />
+
+        <Well className="mt-18">
+          <FormProvider {...methods}>
+            <form>
+              <Heading icon={BeakerIcon} size="h6" className="mt-0">
+                {intl.formatMessage({
+                  defaultMessage: "Testing a new table search",
+                  id: "wqzCx3",
+                  description: "Title for a note around testing new search",
+                })}
+              </Heading>
+              <p className="mb-4.5">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "This table has changed. We are testing a new algorithm to find and display better results. Let us know if you encounter any problems with his table.",
+                  id: "v1yZ/G",
+                  description: "Body of a note around testing new search",
+                })}
+              </p>
+              <Checkbox
+                id="oldSearch"
+                label={intl.formatMessage({
+                  defaultMessage: "Use the old table search",
+                  id: "DDnn7i",
+                  description: "Check box to toggle new search",
+                })}
```

Comment:
Minor quibble. 

I think this should default to checked. Normally experimental features tend to be opt-in right? Defaulting to checked has the table search the way the user is familiar with as a default.